src/app/static/src/app/components/figures/types.ts

```diff
@@ -0,0 +1,31 @@
+// TODO these should all be rolled into the mintr schemas and derived from generated types
```

Comment:
Is there a ticket for that?

Reply:
added https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-1574